pkg/sentry/execution/client.go

```diff
@@ -297,6 +301,130 @@ func (c *Client) BatchCallContext(ctx context.Context, method string, params []i
 	return results, nil
 }
 
+// hexToDecimalString converts a hexadecimal string (with or without 0x prefix) to a decimal string.
+// Returns an error if the hex string is invalid.
+func hexToDecimalString(hexStr string) (string, error) {
```

Comment:
This is ignorant question, but would `hex.EncodeToString()` do what we need over this?

Reply:
Yeah lets use that lmao

Reply:
sorry, as you were https://go.dev/play/p/JqKIES2hzwL